src/utilities/regexp/ISO.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable
+  max-len,
+  prefer-named-capture-group,
+*/
+
```

Comment:
These lines were specific to the language tag regexp. We don't need them for the ISO regexp.

```suggestion
```